src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/NetCoreApp3CompatModeTests.cs

```diff
@@ -22,6 +22,36 @@ public NetCoreApp3CompatModeTests(SingleFileSharedState fixture)
             sharedTestState = fixture;
         }
 
+        [InlineData("./extract")]
+        [InlineData("../extract")]
+        [InlineData("extract")]
+        [InlineData("extract\\foo")]
```

Comment:
Are we intentionally testing both `/` and `\`? If so, might warrant a comment and also doing it in the non-compat-mode extraction test.